src/lib/loaders/loaders_without_authentication/geodata.ts

```diff
@@ -0,0 +1,21 @@
+import { compact } from "lodash"
+import { TCity } from "schema/v2/city"
+import factories from "../api"
+
+/** The cities data may have nulls in it */
+type Response = (TCity | null)[]
+
+export default (opts) => {
+  const {
+    geodataLoaderWithoutAuthenticationFactory: geodataLoader,
+  } = factories(opts)
+
+  return {
+    geodataCitiesLoader: () =>
+      geodataLoader<Response>("partner-cities/cities.json")().then(compact),
+    geodataFeaturedCitiesLoader: () =>
+      geodataLoader<Response>("partner-cities/featured-cities.json")().then(
+        compact
+      ),
```

Comment:
Awkward way to do the compaction here. Not sure why this data might have nulls in it but:

![](https://static.damonzucconi.com/_capture/Y0BH8kVBdXA5.png)